Assets/Editor/ModelInspector.cs

```diff
@@ -189,34 +222,40 @@ public override void OnPreviewSettings()
 
             DrawMeshPreviewToolbar();
         }
-
+        
         private void DoPopup(Rect popupRect, string[] elements, int selectedIndex, GenericMenu.MenuFunction2 func, bool[] disabledItems)
         {
             GenericMenu menu = new GenericMenu();
             for (int i = 0; i < elements.Length; i++)
             {
                 var element = elements[i];
                 
-                if(disabledItems[i])
+                if (disabledItems == null || disabledItems[i])
                     menu.AddItem(new GUIContent(element), i == selectedIndex, func, i);
                 else
                     menu.AddDisabledItem(new GUIContent(element));
             }
             menu.DropDown(popupRect);
         }
-        
+
         private void SetUVChannel(object data)
         {
             int popupIndex = (int)data;
             if (popupIndex < 0 || popupIndex >= m_Settings.availableUVChannels.Length)
                 return;
 
             m_Settings.activeUVChannel = popupIndex;
-            
-            if(m_Settings.displayMode == DisplayMode.UVLayout || m_Settings.displayMode == DisplayMode.UVChecker)
+
+            if (m_Settings.displayMode == DisplayMode.UVLayout || m_Settings.displayMode == DisplayMode.UVChecker)
                 m_Settings.activeMaterial.SetInt("_UVChannel", popupIndex);
         }
-        
+
+        void DestroySkinnedMeshRendererAndRelatedObjects()
```

Comment:
Minor: `DestroySkinnedMeshRendererAndRelatedObjects` function name no longer reflects what it does